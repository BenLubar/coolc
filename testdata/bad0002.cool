class Main() extends A() {
}
class A() extends B() {
}
class B() extends A() {
}
