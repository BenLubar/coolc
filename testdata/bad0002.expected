testdata/bad0002.cool:3:7: class heirarchy loop: A
testdata/bad0002.cool:5:7: class heirarchy loop: B
